reports/technical_report/5_conclusions.md

```diff
@@ -0,0 +1,62 @@
+
+Our analysis of the evolution of normalised claimant counts at the council area level and their correlation with other secondary data suggests that wealthier local economies with better educated workforces have suffered a stronger shock from Covid-19. At the same time, these areas are likely to have access to the resources and capabilities to overcome this shock, not least the skills to diversify into industries that are less exposed to the pandemic. Many of the digitalised and knowledge intensive sectors that our analysis suggests are less exposed and more able to move into new markets are more important in those areas. An important challenge for policymakers will be to put in place interventions to support industries with less diversification opportunities whitstand the shocks of the pandemic while, at the same time, boosting their ability to transition into new industries while mitigating potential increases in inequality (many of the industries that are highly exposed to the pandemic and have low diversification opportunities are low skill, low salary and low productivity). This will be important for boosting Scottish' local economies resilience to Covid-19 in the short term, and to other future shocks linked to Brexit, automation and the environmental transition that may be coming down the line. The analysis that we have undertaken here underscores the potential of novel, high granularity data sources for informing such policies.
+
+
+## Next steps
+
+There are several avenues that we could follow in order to improve and expand our analysis of sectoral exposure and diversification from Covid-19, and to visualise its results.
+
+
+### Aggregated analysis
+
+Our industry vocabularies are based on the business website descriptions of all companies in the UK, which might miss products and services that Scotland specialises on within particular industries. We could address this by re-building the business vocabularies using only the descriptions from Scottish companies. One potential issue with this approach is that it would reduce the sample sizes we use to create industrial vocabularies potentially introducing noise in our results. In a similar vein, and as mentioned before, we could generate industrial vocabularies at a higher level of SIC resolution in order to capture more accurately the evolution of exposure in detailed SIC industries.
+
+We have extracted Google Search trend data for all the UK under the assumption that these trends will be broadly aligned with the evolution of the situation in Scotland. We could however re-run our queries to extract search trends at the level of Scotland or even sub-national geographies. This might be particularly relevant for the analysis of exposure to Covid-19 in sectors with a strong level of local consumption such as for example food and beverage service activities, as well as differences between parts of the country under different social distancing restrictions.
```

Comment:
Note: requires methodological R&D to be able to put different geographies on the same absolute scale (GTAB doesn't do this)